traits.yml

```
@@ -45,30 +58,123 @@ traits:
               - `"seq003"` - for a sequence in a hierarchy
 
           qualifiedName:
             type: string
-            description: |
-              An unambiguous, humanised name for the entity.
+            description: >
+              An unambiguous, humanized name for the entity.
+
 
               The display name may want to consider the host, and any other
               relevant Context information to form a display name for an
               entity that can uniquely identify the entity in that context.
 
+
               For example:
               - `"dive / build / cuttlefish / model / v1"` for a version
                   of an asset in an 'open recent' menu.
               - `"Sequence 003 [ Dive / Episode 1 ]"` for a sequence in
-                 an hierarchy as a window title.
+                 a hierarchy as a window title.
+  lifecycle:
+    description: >
+      Traits that are concerned with describing aspects of the lifecycle
+      of an entity, such as versioning.
+    members:
+      Version:
+        description: >
+          Describes a specific version of an entity.
+
+
+          We define a version as a revision or iteration of what is
+          otherwise the same logical entity. Versions are created when
+          new data is published to an existing entity reference of the
+          same trait set.
+
+
+          There is no requirement for version to be a singular atomic
+          series, managers may wish to support "meta versions", such as
+          'vLatest' or similar, or provide multiple paralell versioning
+          streams.
+
+
+          This trait can be used in several places:
+           - When resolved, the manager should provide information about
+             version of the referenced entity. This trait should only be
+             imbued if the target entity is considered versioned by the
+             manager, and it can populate the stableTag property.
+           - When responding to managementPolicy for an entity trait set,
+             the manager should imbue this trait if that type of entity
+             is version managed by the manager (not all managers version
+             all type of entity).
+           - When used within a relationship query, this trait indicates
+             that the returned entites should be constrained to other
+             versions of the locical entity targeted by the reference.
+        usage:
+          - entity
+          - relationship
+          - managementPolicy
+        properties:
+          tag:
+            type: string
+            description: >
+              A entity-unique, unabiguous identifier for the specific
+              version of an entity targeted by the specific reference.
+
+
+              Examples of version tags include "1", "v2", "latest".
+
 
+              If the reference itself does not contain a version
+              specifier, and relies on dynamic behaviour this should be
```

[consider][minor] read-cadence
```suggestion
              specifier and relies on dynamic behaviour, then this should be
```